### DISCUSSION TITLE
understand MqttClient::LastError() int

### DISCUSSION THREAD

**Discussion by certabitmh** [aws/aws-iot-device-sdk-cpp-v2#697] [Q&A] [answered]
when calling LastError() method of https://aws.github.io/aws-iot-device-sdk-cpp-v2/class_aws_1_1_iot_1_1_mqtt_client.html#a7019e052869b4b121aed212f2b3347a4 , I get 5132 (int).

How is this integer mapped to a meaningful error description? Found enum aws_common_error in aws/comon/error.h but still have no clue on that.

Thanks

**Comment by bretambrose** [comment, marked as answer]
https://github.com/awslabs/aws-crt-cpp/blob/v0.23.0/include/aws/crt/Api.h#L205 will give you a brief error description

**Reply by certabitmh** [reply]
awesome, thanks!
